torchvision/ops/dice_loss.py

```diff
@@ -0,0 +1,71 @@
+import torch
+import torch.nn.functional as F
+
+from ..utils import _log_api_usage_once
+
+
```

Comment:
Plz cite kornia. As we adapter their implementation. And I used it for validation too.

Reply:
Is there a specific format we follow? Or do I add it to the doc string as a reference?

Reply:
You can add a comment like this
https://github.com/pytorch/vision/blob/main/torchvision/ops/boxes.py#L276